CautionaryAlertsListener/serverless.yml

```diff
@@ -8,25 +8,20 @@ provider:
     apiGateway: true
   vpc: ${self:custom.vpc.${opt:stage}}
   stage: ${opt:stage}
-  account: ${opt:account}
   region: eu-west-2
 
 package:
-  artifact: ./bin/release/netcoreapp3.1/base-listener.zip
+  artifact: ./bin/release/netcoreapp3.1/cautionary-alerts-listener.zip
 
 functions:
-  BaseListener:
+  CautionaryAlertsListener:
     name: ${self:service}-${self:provider.stage}
-    handler: BaseListener::BaseListener.SqsFunction::FunctionHandler
+    handler: CautionaryAlertsListener::CautionaryAlertsListener.CautionaryAlertsListener::FunctionHandler
     role: lambdaExecutionRole
     environment:
-      ### Re-use the same parameter as used by web api applications
-      ENVIRONMENT:  ${ssm:/housing-tl/${self:provider.stage}/aspnetcore-environment}
+      CONNECTION_STRING: Host=${ssm:/uh-api/${self:provider.stage}/postgres-hostname};Port=${ssm:/uh-api/${self:provider.stage}/postgres-port};Database=uh_mirror;Username=${ssm:/uh-api/${self:provider.stage}/postgres-username};Password=${ssm:/uh-api/${self:provider.stage}/postgres-password}
```

Comment:
You need to add the other environment variables as well, such as tenureApiToken and tenureApiUrl